spec/requests/pingometer_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+
+RSpec.describe 'Pingometer Webhook', type: :request do
+  it 'is persisted appropriately in the database' do
+    monitor_id = 'b74014410cc1236a3d0h7400'
+
+    # Expect a pre-existing monitor
+    web_service = WebService.create pingometer_id: monitor_id,
+      raw_monitor_data: {
+        'hostname': 'test.com'
+      }
+
+    # Expect a screenshot
+    expect(Browserstack).to receive(:screenshot) {
+      File.new(Rails.root.join('spec', 'fixtures', 'screenshot.png'), 'r')
+    }.at_least(:once)
+
+    pingometer_data = {
+      monitor_id: monitor_id,
+      monitor_host: 'pingometer.com',
+      monitor_status: 'down',
+      utc_timestamp: '2015-03-10T13:13:19' # Pingometer uses ISO8601 but without the Z
+    }
+    post '/hooks/event', pingometer_data.to_json, 'Content-Type' => 'application/json'
+    expect(response).to be_success
+
+    # WebServices should found or be created if they don't exist
+    # web_service = WebService.find_by_pingometer_id monitor_id
+    # expect(web_service).to_not be_nil
+
+    # A webservice without an open incident should result in
+    # the creation of a new incident
```

Comment:
I guess I was getting at was: is it worth testing the difference between “up event w/ no pre-existing incidents for the monitor” vs. “up event in repetition?” It may not be.